astroquery/mast/cutouts.py

```diff
@@ -70,21 +71,37 @@ def _parse_cutout_size(size):
                       InputWarning)
 
     # Getting x and y out of the size
+
     if np.isscalar(size[0]):
         x = size[1]
         y = size[0]
         units = "px"
+        limit_reached = True if ((x >= 30) or (y >= 30)) else False
```

Comment:
The following is equivalent:
```suggestion
        limit_reached = x >= 30 or y >= 30
```
But based on the pull request description the following should be used:
```suggestion
        limit_reached = x > 30 or y > 30
```